### PR TITLE
Remove some node tests from exclusion

### PR DESCRIPTION
**Pull request by hariharans29**
**Description**: Remove some node tests which previously had no implementations (but which have implementations now)

**Motivation and Context**
Get ORT ready for release

### CODE REVIEW

File: onnxruntime/test/onnx/main.cc [hunk 1]
```diff
@@ -449,25 +447,12 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"resize_upsample_sizes_nearest", "not implemented yet"},
```
Comment (hariharans29) [inline onnxruntime/test/onnx/main.cc]: Resize should be covered by #2088

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 2]
```diff
@@ -148,19 +138,15 @@ def create_backend_test(testname=None):
                                  '^test_resize_upsample_sizes_nearest_ceil_half_pixel_cpu',
                                  '^test_resize_upsample_sizes_nearest_cpu',
                                  '^test_resize_upsample_sizes_nearest_floor_align_corners_cpu',
-                                 '^test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu',
-                                 '^test_sequence_*',
-                                 '^test_scatter_.*',
-                                 '^test_onehot_.*',
-                                 '^test_edge_pad_cpu.*',  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
```
Comment (hariharans29) [inline onnxruntime/test/python/onnx_backend_test_series.py]: Moved Pad to the section dedicated to missing type support based test exclusion

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 3]
```diff
@@ -172,6 +158,10 @@ def create_backend_test(testname=None):
                                       'test_unsqueeze_negative_axes_cpu',
                                       'test_constant_pad_cpu']
 
+       if c2.supports_device('MKLDNN'):
+            current_failing_tests += ['^test_range_float_type_positive_delta_expanded_cpu',
```
Comment (hariharans29) [inline onnxruntime/test/python/onnx_backend_test_series.py]: The "expanded" version of the composed Range op (containing the function body as a graph) has an issue in builds with mkldnn enabled. But the single node 'Range' tests themselves pass.

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 4]
```diff
@@ -148,19 +138,15 @@ def create_backend_test(testname=None):
...
-                                 '^test_reflect_pad_cpu.*'  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
+                                 '^test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu'
         ]
 
         # Example of how to disable tests for a specific provider.
         # if c2.supports_device('NGRAPH'):
         #    current_failing_tests.append('^test_operator_repeat_dim_overflow_cpu')
         if c2.supports_device('NGRAPH'):
             current_failing_tests += ['^test_clip.*',
+                                      '^test_qlinearconv_cpu.*',
```
Comment (hariharans29) [inline onnxruntime/test/python/onnx_backend_test_series.py]: This was excluded for NGraph in main.cc.

File: onnxruntime/test/onnx/main.cc [hunk 5]
```diff
@@ -449,25 +447,16 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
...
-      {"onehot_negative_indices", "OneHot(11) not implemented yet"},
       {"bitshift_right_uint8", "BitShift(11) uint8 support not enabled currently"},
       {"bitshift_right_uint16", "BitShift(11) uint16 support not enabled currently"},
       {"bitshift_left_uint8", "BitShift(11) uint8 support not enabled currently"},
       {"bitshift_left_uint16", "BitShift(11) uint16 support not enabled currently"},
-      {"reflect_pad", "test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests"},
-      {"edge_pad", "test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests"},
+      {"reflect_pad", "Pad(11) int32 support not enabled currently"},
+      {"edge_pad", "Pad(11) int32 support not enabled currently"},
+      {"onehot_without_axis", "OneHot(11) not implemented yet"},
```
Comment (hariharans29) [inline onnxruntime/test/onnx/main.cc]: OneHot remains unimplemented for opset-11

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 6]
```diff
@@ -111,55 +113,24 @@ def create_backend_test(testname=None):
...
         current_failing_tests = [#'^test_cast_STRING_to_FLOAT_cpu',  # old test data that is bad on Linux CI builds
-                                 '^test_qlinearconv_cpu',
-                                 '^test_gru_seq_length_cpu',
-                                 '^test_dynamicquantizelinear_expanded.*',
-                                 '^test_dynamicquantizelinear_max_adjusted_expanded.*',
-                                 '^test_dynamicquantizelinear_min_adjusted_expanded.*',
-                                 '^test_top_k.*',
                                  '^test_unique_not_sorted_without_axis_cpu', # bad expected data. enable after https://github.com/onnx/onnx/pull/2381 is picked up
                                  '^test_mod_float_mixed_sign_example_cpu', #onnxruntime::Mod::Compute fmod_ was false. fmod attribute must be true for float, float16 and double types
-                                 '^test_shrink_cpu', #Invalid rank for input: x Got: 1 Expected: 2 Please fix either the inputs or the model.
```
Comment (hariharans29) [inline onnxruntime/test/python/onnx_backend_test_series.py]: invalid input shape/rank now result in warnings only (don't error out). So this test can be re-enabled.

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 7]
```diff
@@ -95,7 +95,9 @@ def test_with_types_disabled_due_to_binary_size_concerns_filters():
     filters = ['^test_bitshift_right_uint16_cpu',
                '^test_bitshift_right_uint8_cpu',
                '^test_bitshift_left_uint16_cpu',
-               '^test_bitshift_left_uint8_cpu']
+               '^test_bitshift_left_uint8_cpu',
+               '^test_edge_pad_cpu.*',
+               '^test_reflect_pad_cpu.*']
```
Comment (skottmckay) [inline onnxruntime/test/python/onnx_backend_test_series.py]: Don't think you need to wildcard the end of these test names. Probably doesn't matter too much as this is temporary given [https://github.com/microsoft/onnxruntime/pull/2179](https://github.com/microsoft/onnxruntime/pull/2179) is tagged with the 1.0 milestone.

File: onnxruntime/test/python/onnx_backend_test_series.py (hunk 4 above)
Comment (skottmckay) [reply onnxruntime/test/python/onnx_backend_test_series.py]: nit: 3 tabs at end of line, and the .* wildcarding at the end isn't needed is it?

---
In reply to: [336303319](https://github.com/microsoft/onnxruntime/pull/2168#discussion_r336303319) [](ancestors = 336303319)

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 8]
```diff
@@ -171,6 +142,10 @@ def create_backend_test(testname=None):
                                       'test_unsqueeze_negative_axes_cpu',
                                       'test_constant_pad_cpu']
 
+        if c2.supports_device('MKLDNN'):
+            current_failing_tests += ['^test_range_float_type_positive_delta_expanded_cpu',
+                                      '^test_range_int32_type_negative_delta_expanded_cpu']		
+ 									  
         if c2.supports_device('OPENVINO_GPU_FP32') or c2.supports_device('OPENVINO_GPU_FP16'):
```
Comment (skottmckay) [inline onnxruntime/test/python/onnx_backend_test_series.py]: nit: tabs here and at end of previous line

File: onnxruntime/test/python/onnx_backend_test_series.py (hunk 7 above)
Comment (hariharans29) [reply onnxruntime/test/python/onnx_backend_test_series.py]: yes you are right. will let #2179 remove it altogether from exclusion.

File: onnxruntime/test/python/onnx_backend_test_series.py (hunk 8 above)
Comment (hariharans29) [reply onnxruntime/test/python/onnx_backend_test_series.py]: Fixed.

File: onnxruntime/test/python/onnx_backend_test_series.py (hunk 4 above)
Comment (hariharans29) [reply onnxruntime/test/python/onnx_backend_test_series.py]: Fixed tabs and removed wildcarding. This seems to have been just copied over.

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 9]
```diff
@@ -111,55 +113,24 @@ def create_backend_test(testname=None):
...
-                                 '^test_resize_upsample_sizes_nearest_ceil_half_pixel_cpu',
-                                 '^test_resize_upsample_sizes_nearest_cpu',
-                                 '^test_resize_upsample_sizes_nearest_floor_align_corners_cpu',
-                                 '^test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu',
-                                 '^test_sequence_*',
-                                 '^test_scatter_.*',
-                                 '^test_edge_pad_cpu.*',  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
-                                 '^test_reflect_pad_cpu.*'  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
-        ]
+                                 '^test_resize_downsample_scales_cubic_A_n0p5_exclude_outside_cpu',  # not implemented yet
```
Comment (askhade) [inline onnxruntime/test/python/onnx_backend_test_series.py]: Can you please remove '^test_resize_downsample_scales_cubic_A_n0p5_exclude_outside_cpu',  and '^test_resize_upsample_scales_cubic_A_n0p5_exclude_outside_cpu',  
from this list... I removed it from main.cc but missed this file

Comment (hariharans29) [reply onnxruntime/test/python/onnx_backend_test_series.py]: Removed.

**Pull request merged**